diff -ruNa gdb-7.2.orig//gdb/configure gdb-7.2/gdb/configure
--- gdb-7.2.orig//gdb/configure	2011-08-05 16:50:09.024372012 +0900
+++ gdb-7.2/gdb/configure	2011-08-05 16:52:59.121372001 +0900
@@ -9743,7 +9743,7 @@
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for readline_echoing_p" >&5
 $as_echo_n "checking for readline_echoing_p... " >&6; }
   save_LIBS=$LIBS
-  LIBS="$LIBS $READLINE"
+  LIBS="$READLINE $LIBS"
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
